### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by eduardo-plentific**
Added $SHELL to init.sh to stop terminal from closing.
Changed README.md to a proper version.
Deleted generated input/output folders and updated .gitignore.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,58 @@
...
+![](https://user-images.githubusercontent.com/7695861/113501088-3590ad00-9523-11eb-81b3-448fd735375b.jpg?raw=true)
 
 Part of the upcoming tutorial from my blog [poissonisfish](https://poissonisfish.com).
 
-## Instructions
+# Instructions
 
-### Set up virtualenv
-In a working directory of your choice, run `pip env python3 -m venv <ENV_NAME>`, then `source <ENV_NAME>/bin/activate`. Finally, install packages using `pip install -r requirements.txt`.
+- [x] Install Python version > 3.9.0. If you don't have it, you can find it [here](https://www.python.org/downloads/).
```
Comment (monogenea) [inline README.md]: I used Python 3.7.6. If you were able to use `requirements.txt` with 3.9.0 I propose rephrasing to "Install Python version >= 3.7"

File: .gitignore [hunk 2]
```diff
@@ -1,4 +1,6 @@
 *.MOV
 objtrack_env
 .vscode
-yolov3
\ No newline at end of file
+yolov3
+input
```
Comment (monogenea) [inline .gitignore:5]: I would rather keep `input/` and `output/` and remove them from `.gitignore` to provide an example along with the code

File: README.md [hunk 3]
```diff
@@ -1,24 +1,58 @@
-# Detection and tracking in Python
+# Object detection and tracking in Python
 
-<p align="center">
-  <img src="https://user-images.githubusercontent.com/7695861/113501088-3590ad00-9523-11eb-81b3-448fd735375b.jpg?raw=true" alt="Example frame"/>
```
Comment (monogenea) [inline README.md:4]: I would prefer to keep the centre-aligned image

**Review by monogenea — CHANGES_REQUESTED** [review]
Overall I am happy with your changes but please address my comments on your latest commit. 🖖 thanks for contributing

File: README.md (hunk 3 above)
Comment (eduardo-plentific) [reply README.md:4]: HTML tags are not Markdown compliant. README.md is a markdown file and Markdown does not care for positioning or styling, hence the change.

File: .gitignore (hunk 2 above)
Comment (eduardo-plentific) [reply .gitignore:5]: Generally control flow objects (`./input` in this case) are not committed, specially output code/results (`./output` in this case). It also grows the size of the code base to download. GitHub provides a section so you can show a demo of your code.

File: README.md (hunk 1 above)
Comment (eduardo-plentific) [reply README.md]: Fixed.

**Pull request merged**